stories/wc-helpers.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  ArgsTable,
+  Stories,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs/blocks';
+
+import componentDocs from 'web-components/component-docs.json';
```

Comment:
This json file is a result of our `web-components` build, and we are using it here to add "auto-generated" documentation into storybook using these helper functions.